tests/test_attentions.py

```diff
@@ -81,7 +81,7 @@ def noop(x):
     return multi_head
 
 
-@pytest.mark.parametrize("attn_dropout", [0.0, 0.3])
+@pytest.mark.parametrize("attn_dropout", [0.3])
```

Comment:
0 and 0.3 are different codepaths (0 will skip dropout completely for most implementations), I think we should keep both for safety reasons